app/views/companies/index.html.haml

```diff
@@ -14,14 +14,24 @@
       #{t('.h_companies_listed_below')}
       #{t('.how_to_search')}
 
-  %table
+    .panel.panel-default.panel-success{ style: 'padding-left: 5px; padding-right: 5px;' }
```

Comment:
I removed ```.panel-success``` - we still have a border but it is now light gray and less obtrusive.